src/content/app/genome-browser/components/track-panel/trackPanelConfig.ts

```diff
@@ -52,9 +55,19 @@ export type TrackStates = {
   };
 };
 
+// we might need to add other types for variants and regulation?
+export type GeneTrackStates = {
+  [objectId: string]: {
+    gene: {
```

Comment:
The `gene` field inside a focus object doesn't make sense. The focus object is a gene in this case.